expr-eval/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for expr-eval 1.0
+// Project: https://github.com/silentmatt/expr-eval#readme
+// Definitions by: Connor Peet <https://github.com/connor4312>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace exprEval {
+    export class Parser {
+        constructor(options?: { allowMemberAccess?: boolean });
+        parse(expression: string): Expression;
+        evaluate(expression: string, values?: any): number;
+        static parse(expression: string): Expression;
+        static evaluate(expression: string, values?: any): number;
+    }
+
+    export interface Expression {
+        simplify(values?: any): Expression;
```

Comment:
Hm, okay. It's also (theoretically) able to handle [string expressions](https://github.com/silentmatt/expr-eval/blob/master/test/parser.js#L114), but I'll add that in as well. Better than `any` I suppose! Give me a few minutes to test it in our code to make sure it works...